en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -1,118 +1,163 @@
+2.  **User store health checker** - This checker iterates through configured user stores of the super tenant domain and attempts to invoke the `isExistingUser` method to check whether a failure occurs.
 
 !!! note
-**Note:** This API is only supported for WSO2 Carbon products that are running on Java 8 or a later version.
-
-
-There are three health checkers available by default:
-
-1.  **Data sources health checker** - This checker goes through the data sources that are configured in the `master-datasources.xml` file and checks if the active connection count surpasses a healthy percentage limit (e.g., 80%) of the maximum allowed connections count. This checker also tests the connection from each data source to see whether the connection is successful.
-2.  **Server startup health checker** - This checker uses the ServerAdmin service to check if the server status is RUNNING.
-3.  **Super tenant user store health checker** - This checker iterates through configured user stores of the super tenant domain and attempts to invoke the `isExistingUser` method to check whether a failure occurs.
+    This API is only supported for WSO2 API Manager that runs on Java 8 or a later version.
 
 ### Deploying the API
 
-!!! warning
-This API is supported by default from WSO2 Identity Server 5.7.0 onwards. It is available by default for WSO2 IS versions 5.5.0 and 5.6.0 only as a WUM update. For more information on how to update using WUM, see [Getting WSO2 Updates](https://docs.wso2.com/display/ADMIN44x/Getting+WSO2+Updates) documentation.
+This section guides you through deploying the Carbon Health Check components in WSO2 API Manager
 
-**If you are using a WSO2 product version that supports this feature by default (either in a fresh pack or via a WUM update), skip the instructions in this section and proceed **to the [configuring the API](#MonitoringServerHealth-ConfiguringtheAPI) section.**
+1. Fork the following GitHub repository.
 
+     ```https://github.com/wso2/carbon-health-check```
+     
+2. Navigate to the place where you want to clone the repo and clone the forked repository. 
 
+     ```git clone https://github.com/[git-username]/carbon-health-check```  
+   
+3. Forked Repository is referred to as `<HEALTH_CHECK_HOME>` in this document. Navigate to `<HEALTH_CHECK_HOME>` and build the product.
+    
+      ```mvn clean install```
+      
+4.  Rename `org.wso2.carbon.healthcheck.api.core-<version-number>-SNAPSHOT.jar` found in the `<HEALTH_CHECK_HOME>/components/org.wso2.carbon.healthcheck.api.core/target` directory as `org.wso2.carbon.healthcheck.api.core-<version-number>.jar`  and paste it in the `<APIM_HOME>/repository/components/dropins` directory.
```

Comment:
why have we not included this into the product?

Reply:
@pubudu538 Could you please answer this ?

Reply:
We can consider adding this to the product. @tharindu1st